sale_mrp_link/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Alex Comba - Agile Business Group
+# Copyright 2016-2018 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sale MRP Link',
+    'summary': 'Show manufacturing orders generated from sales order',
+    'version': '10.0.1.0.0',
+    'development_status': 'Beta',
+    'category': 'Sales Management',
+    'website': 'https://github.com/OCA/sale-workflow',
+    'author': 'Agile Business Group, Akretion, '
+              'Odoo Community Association (OCA)',
```

Comment:
I think authorship is normally in order of importance.
Also I do not think you should add an author for a standard migration. Adding it in contributor yes, but author?
Anyway, other opinions are welcome on the subject.

Reply:
@florian-dacosta, no problem to change the order of authorship. Please note this is not just a standard migration: I added a new code and modified the original one. This is the reason I chose to add my company to author.